src/unittest/test_settings.cpp

```diff
@@ -94,7 +96,9 @@ const char *TestSettings::config_text_after =
 	"	spread = (250,250,250)\n"
 	"}\n"
 	"zoop = true\n"
-	"coord2 = (1,2,3.3)\n"
+	"coord2 = (1,2,3.25)\n"
+    "coord_invalid = (1,2,3\n"
```

Comment:
please add tests like this to ensure how wrong things (and very stupid) works:
*  (wtf, is, this)
* (1, wtf, 3)
* ()
* (1, 2)
* (1)

Reply:
yeah, more error cases are probably good